katana/wui/core/static/core/js/main.js

```diff
@@ -836,6 +836,150 @@ var katana = {
 		},
 
 	},
+
+	openFileExplorer(heading, callBack_on_accept, callBack_on_dismiss){
+	    if(!heading || heading == "" || heading == undefined){
+	        heading = "Select a file"
+	    }
+
+	    if(callBack_on_accept == undefined){
+	        callBack_on_accept = false;
+	    }
+
+	    if(callBack_on_dismiss == undefined){
+	        callBack_on_dismiss = false;
+	    }
+
+	    $.ajax({
```

Comment:
can you use the katana.templateAPI.get method instead of $.ajax

Reply:
Addressed.